src/ansys/fluent/core/systemcoupling.py

```diff
@@ -72,12 +73,16 @@ def solve(self) -> None:
 
     def __get_syc_setup(self) -> dict:
         def get_scp_string() -> str:
-            """Get SCP file contents in the form of the XML string."""
-            fluent_cwd = self._solver.connection_properties.cortex_pwd
-            scp_file_name = os.path.join(fluent_cwd, "fluent.scp")
+            """Get the SCP file contents in the form of an XML string."""
+
+            scp_file_name = "fluent.scp"
             self._solver.setup.models.system_coupling.write_scp_file(
                 file_name=scp_file_name
             )
+
+            # download the file locally in case Fluent is in a container
+            self._solver.download(scp_file_name)
```

Comment:
Note this is going to raise errors when running locally and/or without PyPIM, until we improve this behavior as discussed in the other comment chain